addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -150,7 +150,13 @@
                         <h1 class="d-flex flex-row justify-content-between">
                             <field name="priority" widget="priority" class="me-3"/>
                             <field name="name" class="o_task_name text-truncate" placeholder="Task Title..."/>
-                            <field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}"/>
+                            <div class="d-flex justify-content-end o_state_container" attrs="{'invisible': [('active', '=', False)]}">
+                                <field name="state" widget="project_task_state_selection" class="o_task_state_widget"/>
+                            </div>
+                            <div class="d-flex justify-content-start o_state_container w-100 w-md-50 w-lg-25" attrs="{'invisible': [('active', '=', True)]}">
+                                <field name="state" widget="project_task_state_selection" class="o_task_state_widget"/>
+                            </div>
+                            <field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}" invisible="1"/>
```

Comment:
> Why is state displayed twice ? This should be removed right ?

`<field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}" invisible="1"/>
`
We cannot remove it from the stable version, as it would break customizations if someone uses it in XPath in their module.

Reply:
> > Why is state displayed twice ? This should be removed right ?
> 
> `<field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}" invisible="1"/> ` This state is always hidden, and we can’t remove it from the stable version because if it is used in XPath in a module, it will break the customizations.

The problem is the `<field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}" invisible="1"/>` will never be triggered if the xpath is `//field[@name='state']`, so it is a bit useless to let that field.